example/lib/main.dart

```diff
@@ -18,10 +18,9 @@ import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
 import 'package:amplify_flutter/amplify_flutter.dart';
 import 'package:amplify_storage_s3/amplify_storage_s3.dart';
 import 'package:flutter/material.dart';
-import 'package:sample_app/Pages/LoadingPage.dart';
-
-import 'Pages/LandingPage.dart';
+import 'package:sample_app/Pages/loading_page.dart';
 
+import 'Pages/landing_page.dart';
```

Comment:
The Dart analyzer won't complain about the mismatch in casing, but can you update it to match the new casing?

```suggestion
import 'pages/landing_page.dart';
```